cmd/exec/exec.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/gopinath-langote/1build/cmd/config"
 	"github.com/gopinath-langote/1build/cmd/models"
 	"github.com/gopinath-langote/1build/cmd/utils"
+	"github.com/spf13/viper"
```

Comment:
Any reason to introduce this dependency? 

As we are using cobra default flags which is working fine for us.

Reply:
I faced 2 issues  while accessing --quiet flag in exec.go(exec package). So this dependency added.
1) as I added PersistentFlags() to rootCmd (cmd/root.go) , and rootCmd is private identifier,
So rootCmd is not available outside `cmd` pacakge (rootCmd is required to get flag value),
and we need to access flag(--quiet) in `exec` package.
2) Even if we make, rootCmd as global identifier , and when we access  rootCmd from exec.go, Problem of cyclic import arises.

Reply:
@gopinath-langote what do you think of this ?

Reply:
I see. Circular dependency we definitely don't want.

- Can we parse the parameter (flags) in root command pass those to exec?

Reply:
yes, we can pass parameter in rootCmd Run Function (root.go).
like .  _**exec.ExecutePlan(quietFlag, args...)**_
1) we need to update parameters in exec.go functions also.
     a) ExecutePlan().  -> from root.go this function is called
     b) executeAndStopIfFailed(). -> from ExecutePlan() this function is called.
         and this function is called 3 times in exec.go
2) Even in future also, if we need to add more flags like this then parsing flags and passing to function, might become complex, as we need to pass every flag to function. 

@gopinath-langote what do you think ? should I use this implementation of parsing  parameters ?

Reply:
@ashupatil96 Thanks for the detailed explanation. 

As the parsing implementation needs lot of effort and also makes code complex.

Let's go ahead with your implementation using `viper`